src/plugins/common/components/seo-fields/index.jsx

```diff
@@ -32,7 +32,7 @@ const SeoFields = ({ areLinkedImagesAllowed, isEditMode, node }) => {
     }
   };
   return (
-    <Card>
+    <Card className="seo-fields">
```

Comment:
@digitalartcrew do you need this one as well as the other?

Reply:
The purpose of this class was to replicate the BEM css as in other components and it's a generic class to simply identify the component: https://github.com/triniti/cms-js/blob/master/src/components/key-values-field/index.jsx

The other class is so when the component is imported, we only update the css.

Reply:
@digitalartcrew in that case i would name the other class `seo-fields__seo-published-at` since that is the actual name of the field in the schema